passenger_exporter_test.go

```diff
@@ -158,6 +162,7 @@ func TestUpdateProcessIdentifiers(t *testing.T) {
 				Process{PID: "cdf"},
 				Process{PID: "dfe"},
 			},
+			6,
```

Comment:
How have you come up with `6` here?

Reply:
Max processes is controlled by the [`--max-pool-size`](https://www.phusionpassenger.com/library/config/standalone/reference/#--max-pool-size-max_pool_size) setting. By default its value is 6. The idea is that this value will always be greater or equal to the number of processes at any given point, hence making it a good value for setting the size of the `found` array.

The values chosen in tests are arbitrary, but always greater or equal to the number of items in the `input` map and `processes` array of `updateProcessSpec`.

Reply:
👌

Reply:
Ok, that explains why in the code we're checking `if pid == "" {`:

https://github.com/Intellection/passenger-exporter/blob/67ec97b6b8cd953fe9cd54b7cf7256abeef0c08e/passenger_exporter.go#L394-L395

Reply:
For interest sake, it panics on line 377:

https://github.com/Intellection/passenger-exporter/blob/67ec97b6b8cd953fe9cd54b7cf7256abeef0c08e/passenger_exporter.go#L375-L377